optaweb-employee-rostering-standalone/run_tests.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+readonly CONTAINER_RUNTIME=$1
+readonly FRONTEND_PROJECT_PATH=$2
+readonly VERSION_CYPRESS=$3
+readonly APPLICATION_PORT=$4
+
+user_command="--user  $(id -u):$(id -g)"
+
+if [[ "$CONTAINER_RUNTIME" == "podman" ]]
+then
+    user_command="${user_command} --userns=keep-id"
+fi
+
+echo "$CONTAINER_RUNTIME run --network=host \
```

Comment:
Why are you creating a file with this content instead of directly executing it like this?:

```
$CONTAINER_RUNTIME run --network=host \
                          -v $FRONTEND_PROJECT_PATH:/e2e:Z \
                          $user_command \
                          -w /e2e --entrypoint cypress docker.io/cypress/included:$VERSION_CYPRESS \
                          run --project . --config baseUrl=http://localhost:$APPLICATION_PORT
```

Reply:
The same can be done with an echo (with the added benefit of not needing to modify .gitignore to ignore that generated file); change it to an echo + run, or just run, and I'll approve.